tests/dags/common/loader/test_sql.py

```diff
@@ -99,6 +103,16 @@ def create_query_values(
     return ",".join(result)
 
 
+@pytest.fixture
+def mock_dag() -> DAG:
+    return create_pg_timeout_tester_dag()
+
+
+@pytest.fixture
+def mock_task(mock_dag) -> BaseOperator:
+    return mock_dag.get_task("mock_task")
+
```

Comment:
Since these fixtures are used in multiple places, you can actually set up a `conftest.py` file at a higher directory (whichever level in the hierarchy you feel most appropriate for these) which includes the fixtures. Pytest will then manage making them available to all tests within that directory tree! https://docs.pytest.org/en/7.1.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files